src/Components/Patient/SampleTest.tsx

```diff
@@ -131,21 +175,33 @@ export const SampleTest = (props: any) => {
         has_sari: state.form.has_sari,
         is_unusual_course: state.form.is_unusual_course,
         is_atypical_presentation: state.form.is_atypical_presentation,
-        atypical_presentation: state.form.is_atypical_presentation ? state.form.atypical_presentation : undefined,
+        atypical_presentation: state.form.is_atypical_presentation
+          ? state.form.atypical_presentation
+          : undefined,
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)